visidata/macros.py

```diff
@@ -125,15 +125,37 @@ def startMacro(cmdlog):
         vd.status("recording macro; stop recording with `m`")
         vd.macroMode = CommandLogJsonl('current_macro', rows=[])
 
-
 @VisiData.before
 def run(vd, *args, **kwargs):
     vd.macrosheet
 
+class MacrosGuide(GuideSheet):
+    guide = '''# Macros
+Macros allow you to bind a series of commands to a key and then replay those commands within a session by using that keystroke.
```

Comment:
"bind a command sequence to a keystroke or longname, to replay when that keystroke is pressed or the command is executed by longname."  i.e. make clear that it can be either keystroke or longname.